### ISSUE TITLE
Use on my own data

### ISSUE THREAD

**Issue by yoyoNTNU** [ningshuliang/PICTURE#15]
Hello, I am very interested in this paper, but since I am new in this field, I would like to know how to use your program if I want to edit my own photos? What do I need to prepare? Finally, I would like to ask if there is a way to achieve this on Windows? Or is it just for Ubuntu? Thanks.

**Comment by ningshuliang** [comment]
Yes, if you want to edit your own photos, you need to prepare the densepose, parsing map in my style, or you can train a model based on your data. And then go through the inference code, you can get the final results.
